src/app/shared/components/notification/notification.component.scss

```diff
@@ -0,0 +1,42 @@
+$animation-duration: 0.3s;
+$animation-function: ease;
+$animation-fill-mode: forwards;
+
+:host {
+  --gradient-degree: 90deg;
+  --status-color-width: 5%;
+  padding: 10px;
+  border-radius: 10px;
+  box-sizing: border-box;
+  box-shadow: 1px 1px 2px var(--black-color-with-alpha);
+  position: absolute;
+  display: flex;
+  color: var(--black-color);
+  top: 0;
+  left: 0;
+  width: 98%;
+  transform: translateX(500px);
+  transition: top $animation-duration $animation-function;
+  animation: slide-from-right $animation-duration $animation-function $animation-fill-mode;
```

Comment:
I disagree a little bit with that.
I can see the argument for not having `$animation-fill-mode` as a variable since it is only used once.

I think in general that we are using way too few variables and way too many "magic values" in our CSS. Just because it's CSS doesn't mean we shouldn't apply our principles.

But a good example is actually on the line just above the animations: `transform: translateX(500px)`. Unfortunately, I have written the 500 px as a "magic number" quite a few places in the CSS in different components. They should ideally have been a single variable for the `notification-width` so it would be easy to change the width of our notifications without having to do a minor variable hunt to ensure nothing is broken.

The above is sadly not uncommon in our CSS.

The best example I have is that we set padding and margins in _a lot_ of CSS classes. Ideally we would have dedicated classes for these values and instead of having to define them in each and every CSS class, we could just give the CSS class to the HTML element who wants that padding/margin. 
This would ensure that we have the same paddings/margins in our system it would be easy to update it across our system.